### ISSUE TITLE
How to fork a component?

### ISSUE THREAD

**Issue by acidjunk** [workfloworchestrator/orchestrator-ui-library#596]
I am working on a project where I want to rename: "subscriptions" to "notices".

So I did override the side menu, changes the subscriptions page there; and added some translations. 

Problems: crumble path and navigation from subscription detail is quirky.

So I copied the complete `WfoSubscriptionsListPage` page from the lib into my app, and tried to get that running, but failed.
Most pages use some internal import, and not everything I needed was exported correctly. There also seem to be some things that exists more then once (some internal tab.ts files for example)

- [ ] include the components used on the toplevel pages
- [ ] metadata pages nice to have

**Comment by wouter1975** [comment]
@acidjunk, please include specific missing export which is causing the failure, so we can quickly fix your problem. We need to think of a mechanism to safeguard that all applicable exports are indeed done in the NPM library.

**Comment by acidjunk** [comment]
I think the use case is clear:

every top level page could be forked into a custom page.

A while ago I tried to get the "Subscriptions" list page working from inside my own app

So I copied the library code for `WfoSubscriptionsListPage` to my app `pages/notices.tsx`-> and let Webstorm resolve/change the imports to a place inside the lib. For this page I encountered problems with:

- `SubscriptionListItem`
- `subscriptionListTabs`
- `WfoSubscriptionListTab`
- `WfoSubscriptionsList`

They can be found in: '@orchestrator-ui/orchestrator-ui-components/src/components/WfoSubscriptionsList' -> but not in the root export.
